odata-to-abstract-sql.ometajs

```diff
@@ -156,18 +156,24 @@ export ometa OData2AbstractSQL {
 		)
 
 		(	?(!path.options)
-		|	(	?(!path.options.$filter)
-			|	?(method == 'POST' || method == 'PUT-INSERT')
-				InsertFilter(path.options.$filter, resource, bindVars):subQuery
-				{_.findIndex(query.extras, {0: 'Values'})}:valuesIndex
-				// Replace the values bind var list with our filtered SELECT query.
-				-> (query.extras[valuesIndex] = ['Values', subQuery.compile('SelectQuery')])
+		|	(	?(method == 'POST' || method == 'PUT-INSERT')
```

Comment:
Is that UPSERT right there? So Pine supports UPSERTS? How come we're not using them?